### PR TITLE
Dashboard (delayed)

### PR DESCRIPTION
**Pull request by JeppesGithub**

### CODE REVIEW

**Comment by mathiaszip** [comment]
.

File: public/login/index.html
Comment (DeeKahy) [inline public/login/index.html:1]: @mantarias @CarlHejlesen take a look at this and check if its right

**Review by DeeKahy — COMMENTED** [review]
does the create user work thing?

**Comment by mathiaszip** [comment]
Yup> does the create user work thing?

**Review by mantarias — APPROVED** [review]

**Pull request merged**